matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/DefaultTimeline.kt

```diff
@@ -389,6 +390,7 @@ internal class DefaultTimeline(
     }
 
     private suspend fun loadRoomMembersIfNeeded() {
+        if (RoomLocalEcho.isLocalEchoId(roomId)) return
```

Comment:
I think this check could be avoided if we store the fact that all the members are known for local rooms.